src/client/auth/auth.proto

```diff
@@ -77,12 +77,9 @@ message ActivateResponse {}
 
 service API {
     rpc Activate(ActivateRequest) returns (ActivateResponse) {}
-
```

Comment:
Could you keep the newlines?  They are to distinguish different logical "sections" of the API.